plugins/platform-ui/src/components/internal/Status.svelte

```diff
@@ -12,7 +12,9 @@
 
 <span>
   {#if status.severity === Severity.ERROR}
-    <Icon icon={ui.icon.Error} clazz="icon-embed" style="fill: #C21F39" />
+    <span style="fill: #C21F39">
+      <Icon icon={ui.icon.Error} clazz="icon-embed" />
+    </span>
```

Comment:
why we do not introduce `fill` (or `color`) property to the `Icon`, which seems to be natural fit?